### PR TITLE
 Add the CPU load status, memory status, disk status in DashBoard

### PR DESCRIPTION
**Pull request by 23931017wu**
### Test
image

### CODE REVIEW

File: grafana-metrics-example/cluster/Apache IoTDB ConfigNode Dashboard v0.14.0.json [hunk 1]
```diff
@@ -1092,6 +1092,282 @@
...
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
```
Comment (SpriCoder) [inline grafana-metrics-example/cluster/Apache IoTDB ConfigNode Dashboard v0.14.0.json:1226]: There are no need to set thresholds in Basic Dashboard.

**Review by SpriCoder — CHANGES_REQUESTED** [review]
PTAL~

File: grafana-metrics-example/cluster/Apache IoTDB ConfigNode Dashboard v0.14.0.json (hunk 1 above)
Comment (23931017wu) [reply grafana-metrics-example/cluster/Apache IoTDB ConfigNode Dashboard v0.14.0.json:1226]: OK

**Review by neuyilan — APPROVED** [review]

**Review by SpriCoder — APPROVED** [review]
LGTM~

**Pull request merged**